book/history.md

```diff
@@ -180,20 +178,19 @@ and the concept on the [alt.hypertext Usenet
 group](https://www.w3.org/People/Berners-Lee/1991/08/art-6484.txt).
 
 ::: {.center}
-![Figure 4: Screenshot of the WorldWideWeb browser[^worldwideweb]](im/worldwideweb.png)
+![Figure 4: Screenshot of the WorldWideWeb browser. ([*Communications of the ACM*][cacm94], August 1994.)](im/worldwideweb.png)
 :::
 
-[^worldwideweb]: <a
-	href="https://dl.acm.org/doi/10.1145/179606.179671">Communications of the
-	ACM</a>, August 1994.
+[cacm94]: https://dl.acm.org/doi/10.1145/179606.179671
 
 [^no-css]: No CSS! No JS! Not even images!
 
 [^more-less-powerful]: For example, the first browser included the concept of an
 index page meant for searching within a site (vestiges of which exist today in
 the “index.html” convention when a URL path ends in /”), and had a WYSIWYG web
 page editor (the “contenteditable” HTML attribute and “html()” method on DOM
-elements have similar semantic behavior, but built-in file saving is gone).
+elements (see [Chapter 15](invalidation.md))
```

Comment:
The copyeditor linked to Chapter 5, but while that *is* the chapter on HTML it doesn't cover contenteditable. I instead linked to Chapter 15, which does.